### ISSUE TITLE
Asset Filter by different fields

### ISSUE THREAD

**Issue by amannb** [snipe/snipe-it#6439]
**Server (please complete the following information):**
 - Snipe-IT Version: v4.6.6 build 3926
 - OS: [e.g. Ubuntu 16.04]
 - Web Server: [e.g. Apache2]
 - PHP Version: 7.1.17-0ubuntu0.17.10.1

**Is your feature request related to a problem? Please describe.**
In the all assets view, I sometimes want to filter it by certain criteria, for example by category, model, manufacturer, branch, etc... 
I would like to be able to do this filtration from the all assets view because at the moment I export it to excel to get a snapshot of our status. 

**Describe the solution you'd like**
In the all assets view, to give options at the top of each column to be able to filter by the available criteria. There is the sort option but I need to filter. 

**Describe alternatives you've considered**
As of now I just export the data to excel of use the custom reports. 

**Additional context**

**Comment by Jarli01** [comment]
This I've also thought would be useful from time to time as there are occasions

**Comment by amannb** [comment]
Yes it is. I would appreciate if this feature would be included

**Comment by amannb** [comment]
Yes I still do need this feature.

**Comment by amannb** [comment]
Yes it is still required.

**Comment by amannb** [comment]
Yes it is still relevant

**Comment by amannb** [comment]
Yes it is still relevant

**Comment by Jarli01** [comment]
Yes.

**Comment by amannb** [comment]
Yes this is still valid

**Comment by amannb** [comment]
Yes it is

**Comment by amannb** [comment]
Yes it is

**Comment by gmintoco** [comment]
just want to +1 this feature as well, would be really useful when look through massive lists of assets

**Comment by gmintoco** [comment]
+1 definitely would find this still useful

**Comment by amannb** [comment]
Yes it is

**Comment by Jarli01** [comment]
Yes

**Comment by amannb** [comment]
Hello Snipe/Snipe-It
Yes is it relevant.

**Comment by Jaspotn0612** [comment]
Yes

**Comment by amannb** [comment]
yes it is

**Comment by hesske** [comment]
Yes, please add filtering capability on the assets and people screens. If Snipe-IT had at least some of the filtering capability that exists in Excel, that would be very helpful.

**Comment by arranreid** [comment]
Relevant. Need this.

**Comment by senorsmile** [comment]
I just found this project, looked at the demo: https://demo.snipeitapp.com/ and immediately noticed there was no way to filter specific columns.  Unfortunately, this makes it less useful than a google sheet for a large enough inventory.  

<blockquote><img src="https://demo.snipeitapp.com/favicon.ico" width="48" align="right"><div><strong><a href="https://demo.snipeitapp.com/">Snipe-IT Asset Management Demo</a></strong></div></blockquote>
